erpnext/crm/doctype/lead/lead.json

```diff
@@ -514,11 +515,10 @@
  "idx": 5,
  "image_field": "image",
  "links": [],
- "modified": "2022-10-13 12:42:04.277879",
+ "modified": "2023-01-24 18:20:05.044791",
  "modified_by": "Administrator",
  "module": "CRM",
  "name": "Lead",
- "name_case": "Title Case",
```

Comment:
@dj12djdjs Why is this removed?

Reply:
@deepeshgarg007 I'm not sure. I only checked Read Only on the title field. Clicked Save.

This actually happened last on my last PR changing a doctype definition. Seems anytime I save a doctype this line gets removed.